code/style.css

```diff
@@ -28,6 +51,14 @@ p {
   border-radius: 16px;
 }
 
+.killer-deck:hover {
```

Comment:
I really appreciate the hover effect, it makes the game more engaging for the user.